### ISSUE TITLE
Error report

### ISSUE THREAD

**Issue by vitepython** [canonical/lxd-ui#787]
```
Error: Could not find macvlan in networkTypeToOptionKey
    at eG (https://192.168.0.200:8443/ui/assets/index-b3f3eaed.js:87:30823)
    at ie (https://192.168.0.200:8443/ui/assets/FormMenuItem-f4d64986.js:1:5988)
    at te (https://192.168.0.200:8443/ui/assets/FormMenuItem-f4d64986.js:1:4865)
    at be (https://192.168.0.200:8443/ui/assets/FormMenuItem-f4d64986.js:1:7865)
    at J (https://192.168.0.200:8443/ui/assets/NetworkForm-34d3abf4.js:1:4879)
    at Zg (https://192.168.0.200:8443/ui/assets/index-b3f3eaed.js:38:19529)
    at B1 (https://192.168.0.200:8443/ui/assets/index-b3f3eaed.js:40:44027)
    at L1 (https://192.168.0.200:8443/ui/assets/index-b3f3eaed.js:40:39763)
    at m$ (https://192.168.0.200:8443/ui/assets/index-b3f3eaed.js:40:39691)
    at Cd (https://192.168.0.200:8443/ui/assets/index-b3f3eaed.js:40:39545)
```

**Comment by edlerd** [comment]
Thank you for reaching out.

I think this has been fixed with [this commit](https://github.com/canonical/lxd-ui/commit/1647a00df3b305d9910d103fee7e232c5ede501b#diff-850090c4ab0bb8a6f5c662830bea10666c074a1f0af2a5a62698c954a3aae848R66), which is currently pending a patch release for LXD 5.21 with UI version 0.8.1.

**Comment by VariableDeclared** [comment]
seeing this issue

**Comment by edlerd** [comment]
While the issue has been fixed in the repo, it is not yet released to the 5.21/stable channel. For the time being you can rely on the 5.21/candidate channel, which already includes the fix.

**Comment by edlerd** [comment]
A fix was released with 5.21.2.
